firefox-ios/firefox-ios-tests/Tests/ClientTests/Wallpaper/Mocks/WallpaperURLSessionMock.swift

```diff
@@ -38,6 +38,14 @@ class WallpaperURLSessionMock: URLSessionProtocol {
         return (data ?? Data(), response ?? URLResponse())
     }
 
+    func data(from urlRequest: URLRequest) async throws -> (Data, URLResponse) {
+        if let error = error {
+            throw error
+        }
+
+        return (data ?? Data(), response ?? URLResponse())
+    }
+
```

Comment:
This isn't called anywhere is it...? 🤔  Can be deleted?

Reply:
Ahh ok, that makes sense then! I think I wasn't looking close enough to see it was in the unit tests. 🙂